Source/WebCore/platform/graphics/skia/FontCascadeSkia.cpp

```diff
@@ -128,13 +128,46 @@ RefPtr<const Font> FontCascade::fontForCombiningCharacterSequence(StringView str
     ++codePointsIterator;
     bool isOnlySingleCodePoint = codePointsIterator == codePoints.end();
 
+    bool shouldForceEmojiFont = false;
+    ResolvedEmojiPolicy emojiPolicy = [&] {
```

Comment:
Looks a bit awkward that the lambda fills in the bool value. Structured binding could encapsulate this better:
```auto [emojiPolicy, shouldForceEmojiFont] = [&] {```. I was originally hoping to get rid of the capture in the lambda, but it uses quite a few other variables from the outer function, so I take back the suggestion ;-)